ui/pages/confirmations/components/confirm/info/typed-sign/typed-sign.tsx

```diff
@@ -46,7 +46,7 @@ const TypedSignInfo: React.FC = () => {
 
   useEffect(() => {
     (async () => {
-      if (!isPermit || !isOrder) {
+      if (!isPermit && !isOrder) {
         return;
```

Comment:
Hey @cryptotavares : we want to return null if signature request is either order or permit type. I think old condition was correct.